t/test01.t

```diff
@@ -254,35 +251,32 @@ subtest 'domain_history' => sub {
         $params->{ipv4} = 0;
 
         # create the test, retrieve its id but we don't run it
-        $rpcapi->job_create( $params );
+        $rpcapi->start_domain_test( $params );
         ( $hash_id ) = $rpcapi->{db}->get_test_request();
 
-        my $res_domain_history;
-        $res_domain_history = $rpcapi->domain_history( $method_params );
-        $domain_history = $res_domain_history->{history};
-        is( scalar( @$domain_history ), 2, 'Only 2 tests should be retrieved' );
+        $test_history = $rpcapi->get_test_history( $method_params );
+        #diag explain( $test_history );
```

Comment:
Just a thought: Would it make sense to call `note` here instead of having a commented out call to `diag`?

Reply:
Good suggestion. I've updated.